test/AutoRest.TestServer.Tests/AutoRest.TestServer.Tests.csproj

```diff
@@ -16,6 +16,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Include="../../src/assets/**/*.cs"/>
+    <Folder Include="Tests" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="../../src/assets/**/*.cs" />
+    <Compile Include="../TestServerProjects/**/*.cs" Exclude="../TestServerProjects/**/Azure.Core.Shared/*.cs" LinkBase="TestServerProjects" />
```

Comment:
This allows us to compile the files as native source to the test project. We can do the same with Samples if we want to write tests for them.